tests/suites/test_suite_cipher.function

```diff
@@ -1184,12 +1059,6 @@ void auth_crypt_tv( int cipher_id, data_t * key, data_t * iv,
                     cipher_id == MBEDTLS_CIPHER_AES_256_KW ||
                     using_nist_kw_padding;
 
-    /****************************************************************
```

Comment:
[This is a comment about another line, but github doesn't let me comment on the proper line.]

Please also remove the comment on lines 1012-1015 that starts with "Do that twice:" as we're now only doing this once :)

Reply:
Done